sdk/communication/tests.yml

```diff
@@ -14,6 +14,31 @@ jobs:
         groupId: com.azure
         safeName: azurecommunicationsms
     ServiceDirectory: communication
+    PostSteps:
```

Comment:
I really like that this customization got moved into the communication yml itself instead of having it conditioned in template.

Reply:
Actually thinking about this a little more those steps look pretty generic but communication is the only service that currently opts into it. It might be worth having these in the template but controlled by a template parameter instead of by pipeline name.

Reply:
It does seem generic but I don't think it's worth keeping them in the template until another service or two comes around and wants to add them. Having a code coverage opt-in is I think outside the current scope of this change.